inventory/sample.aws.example.com.d/inventory/group_vars/all.yml

```diff
@@ -88,43 +131,51 @@ aws_region: <REPLACE WITH VALID AWS REGION>
 
 # If aws_create_vpc is 'true', aws_vpc_id and aws_subnet_id variables will be ignored
 aws_create_vpc: true
```

Comment:
It's my understanding that this doesn't do anything with this PR, so let's make sure there's a comment here stating that it doesn't have any effect setting it to "false" as-is.